### PR TITLE
Add Setup .bat File

### LINKED ISSUE #17: Add Setup Batch File for Windows
By doing so we allow users from windows to use this repository. The script should install npm dependencies, as well as activate a virtual environment and install all dependencies listed in the `requirements.txt` file.

### PR DESCRIPTION
**Pull request by evanugarte**
Resolves #17. Test this out on your machine to ensure it works.

### CODE REVIEW

**Comment by evanugarte** [comment]
@jerrylee17 for some reason `npm install` didn't install the dependencies for me and I had to specify them manually.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .vscode/
 __pycache__/
-modules/
\ No newline at end of file
+modules/
```
Comment (jerrylee17) [inline .gitignore]: change to: `node_modules/`

File: setup.bat [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@ECHO OFF
+
+REM Install client dependencies
+call npm install grpc google-protobuf @grpc/proto-loader
+REM Remove modules (virtual environment folder)
+del modules
```
Comment (jerrylee17) [inline setup.bat]: I get an error that says can't find modules. Maybe you meant `node_modules`?

File: setup.bat [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+REM Install virtualenv
+python -m pip install virtualenv --user
+REM Initialize modules as virtual environment
+virtualenv modules
```
Comment (jerrylee17) [inline setup.bat]: I get the error:
```
'virtualenv' is not recognized as an internal or external command,
operable program or batch file.
```
[Not sure if you've looked at this](https://stackoverflow.com/questions/35950740/virtualenv-is-not-recognized-as-an-internal-or-external-command-operable-prog)

File: setup.bat [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+REM Activate virtual environment
+modules/Scripts/activate.bat
```
Comment (jerrylee17) [inline setup.bat]:
```
'modules' is not recognized as an internal or external command,
operable program or batch file.
```
Assuming this is related to the virtual environment

File: setup.bat [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+REM Install all dependencies
+python -m pip install -r requirements.txt
+REM Deactivate virtual environment
+deactivate
```
Comment (jerrylee17) [inline setup.bat]: Same error, assuming this is fixed with virtual environment getting installed.

**Pull request merged**